packages/drawer/src/DrawerBase.tsx

```diff
@@ -0,0 +1,84 @@
+import Button from '@littlespoon/button'
+import { up, xs } from '@littlespoon/theme/lib/breakpoints'
+import { shadeWhite } from '@littlespoon/theme/lib/colors/token'
+import { family } from '@littlespoon/theme/lib/fonts/primary'
+import { rem } from '@littlespoon/theme/lib/utils'
+import styled from 'styled-components'
+
+import { DrawerProps } from './Drawer'
+
+export const DrawerBase = styled.div<Partial<DrawerProps>>`
+  align-content: stretch;
+  background-color: ${shadeWhite};
+  border: 0;
+  border-radius: ${rem(1.2)} ${rem(1.2)} 0 0;
+  bottom: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  left: 0;
+  margin: 0;
+  max-height: 80%;
+  max-width: none;
+  /* TODO: Update this value */
```

Comment:
Remove todo?

Reply:
Removed in 3f6d3003c5c97af626d79d856b59f1ee93156f06